state/accountsDB.go

```diff
@@ -1263,6 +1263,7 @@ func (adb *AccountsDB) processSnapshotCompletion(
 	defer func() {
 		adb.isSnapshotInProgress.Reset()
 		adb.updateMetricsOnSnapshotCompletion(metrics, stats)
+		close(errChan)
```

Comment:
:+1: this ensures the correct closing of the error chan. Good catch.

Reply:
It is ok. This will be called only after all other launched goroutines will finish the execution. (See `adb.finishSnapshotOperation`)